catalog/model.go

```diff
@@ -87,3 +87,30 @@ func (j *Metadata) Scan(src interface{}) error {
 	}
 	return json.Unmarshal(data, j)
 }
+
+type MinMaxCommit struct {
+	MinCommit CommitID `db:"min_commit"`
+	MaxCommit CommitID `db:"max_commit"`
+}
+
+func (m MinMaxCommit) IsDeleted() bool {
+	return m.MaxCommit != MaxCommitID
+}
+func (m MinMaxCommit) IsTombstone() bool {
+	return m.MaxCommit == TombstoneCommitID
+}
+
+func (m MinMaxCommit) IsCommitted() bool {
+	return m.MinCommit != MaxCommitID
+}
+
+// needed for diff, to check if an entry changed after the lineage commit id
```

Comment:
1. func doc - start with the name of the function 'changedAfterCommit' follow by what it does then who needs it
2. change the function to public